test/api-test.js

```diff
@@ -562,6 +562,12 @@ describe('RippleAPI', function() {
       secret: 'shzjfakiK79YQdMjy4h8cGGfQSV6u'
     };
     assert.doesNotThrow(_.partial(validate.addressAndSecret, goodWallet));
+    assert.throws(_.partial(validate.secret, ''),
+      this.api.errors.ValidationError);
+    //  'passphrase' that starts from 's', like secret
+    assert.doesNotThrow(_.partial(validate.secret, 's!!!'));
```

Comment:
done.
only one thing to note - I was forced to add new method to `Seed`
hope this not a prolem :smiley: